woocommerce/woocommerce
# Update @automattic/tour-kit to 1.1.3 and @automattic/components to 2.1.1

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:






Closes https://github.com/woocommerce/woocommerce/issues/39137.

This PR updates the `@automattic/tour-kit` package to version 1.1.3 to fix the step highlight issue on mobile. I also made the corresponding changes to ensure the tour kit worked as before. Besides, I updated the `@automattic/components` package to version 2.1.1 to to update the `@automattic/tour-kit`

### [Tourkit changelog](https://github.com/Automattic/wp-calypso/blob/trunk/packages/tour-kit/CHANGELOG.md):

<details>
#### 1.1.3

- Fallback reference element to the desktop selector if the mobile selector is not provided

#### 1.1.2

- Added 'onStepView' callback that's called each time a step is mounted 
- Fixed the value of index that is passed to 'onGoToStep', 'onPreviousStep' and 'onNextStep'   https://github.com/Automattic/wp-calypso/pull/88435
</details>




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Set up a fresh store, go through setup wizard and select a US state as the store location and complete setup without installing plugins
2. Open dev tools and enable debug via `window.localStorage.setItem( 'debug', '*')`
3. Use mobile view
4. Go to `/wp-admin/post-new.php?post_type=product&tutorial=true`
5. Confirm it highlights the product title field
6. Click on the Next button
7. Confirm it highlights the product description field and fire the `wcadmin_walkthrough_product_step_completed` event with `step_name = 'product-name'`
8. Go to /wp-admin/admin.php?page=wc-settings&tab=shipping
7. Confirm it highlights the shipping zone column
8. Click on the Next button
9. Confirm it highlights the shipping method column and fire the `wcadmin_walkthrough_settings_shipping_next_click` event with `step_name = 'shipping-zones'`
10. Click on the Back button
11. Confirm it fires the `wcadmin_walkthrough_settings_shipping_back_click` event with `step_name = 'shipping-methods'`

<img width="299" alt="Screenshot 2024-05-03 at 11 45 56" src="https://github.com/woocommerce/woocommerce/assets/4344253/49b91601-d0b7-4258-b678-bed7df80ab19">

<img width="623" alt="Screenshot 2024-05-03 at 11 46 09" src="https://github.com/woocommerce/woocommerce/assets/4344253/bd5ebb24-2948-4f45-aa0c-b2319be85bf3">




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
